game-core/src/test/java/games/strategy/triplea/ui/PropertyFileTest.java

```diff
@@ -1,58 +1,58 @@
-package games.strategy.triplea.ui;

```

Comment:
Good catch.  I searched the rest of the tree for any other files that use CRLF and, other than this file, only came up with Windows batch files, so we should be good everywhere else.